python/lsst/pipe/analysis/coaddAnalysis.py

```diff
@@ -271,10 +271,11 @@ def runDataRef(self, patchRefList, cosmos=None):
 
         # Create and write parquet tables
         if self.config.doWriteParquetTables:
-            tableFilenamer = Filenamer(repoInfo.butler, 'qaTableCoadd', repoInfo.dataId)
             if haveForced:
-                writeParquet(forced, tableFilenamer(repoInfo.dataId, description='forced'), badArray=bad)
-            writeParquet(unforced, tableFilenamer(repoInfo.dataId, description='unforced'), badArray=bad)
+                dataRef_forced = repoInfo.butler.dataRef('analysisCoaddTable_forced', **repoInfo.dataId)
```

Comment:
Are the "\*\*"'s really required (here and elsewhere)?

Reply:
I had tested this by just passing the `dataId` as a positional keyword, and that didn't work, and this did, so that's why I did it.  Perhaps `dataID=repoInfo.dataId` would work, if that is the preferred idiom?

Reply:
Hhhmm...what was the error?  If `dataId=repoInfo.dataId` works I would use that.

Reply:
Doing as suggested; will test to make sure it works.

Reply:
`dataId=repoInfo.dataId` is indeed recommended.  The problem with doing this as a positional parameter is that there is a legacy `level` parameter in the way.

Reply:
Thanks K-T...I'm glad we chose the preferred path!